src/ImageSharp.Web/Processors/ResizeWebProcessor.cs

```diff
@@ -179,46 +177,7 @@ public bool RequiresTrueColorPixelFormat(CommandCollection commands, CommandPars
             }
 
             PointF center = new(coordinates[0], coordinates[1]);
```

Comment:
Probably better to directly create a `Vector2` here, instead of implicitly converting from the `PointF`:
```suggestion
            Vector2 center = new(coordinates[0], coordinates[1]);
```

Reply:
Yeah will update